### PR TITLE
grpc-debug tool

### PR DESCRIPTION
**Pull request by jtolds**
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/283)

### CODE REVIEW

File: examples/grpc-debug/main.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"flag"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"storj.io/storj/pkg/cfgstruct"
+	"storj.io/storj/pkg/provider"
+)
+
+var (
+	targetAddr = flag.String("target", "satellite.staging.storj.io:7777",
```
Comment (egonelbre) [inline examples/grpc-debug/main.go]: don't wrap lines at 80

File: examples/grpc-debug/main.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		"address of target")
+
+	ic provider.IdentityConfig
```
Comment (egonelbre) [inline examples/grpc-debug/main.go]: ic -> identityConfig, there's no real win in shortening this

File: examples/grpc-debug/main.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+func main() {
+	ctx := context.Background()
+	flag.Parse()
+	identity, err := ic.Load()
+	if err != nil {
+		panic(err)
+	}
+	dialOpt, err := identity.DialOption()
```
Comment (egonelbre) [inline examples/grpc-debug/main.go]: dialOpt -> dialOption

File: examples/grpc-debug/main.go (hunk 1 above)
Comment (jtolds) [reply examples/grpc-debug/main.go]: :( but it makes side-by-side diffs so much better.

this is going to be a tough habit for me to break. i'll fix it here though

**Pull request merged**